src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +58,41 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService)
+                injector.getInstance(UserService.class);
+        User user = new User();
+        user.setEmail("bob@gmail.com");
+        user.setPassword("12345");
+        userService.add(user);
+
+        TicketDao ticketDao = (TicketDao)
+                injector.getInstance(TicketDao.class);
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(yesterdayMovieSession);
+        ticket.setUser(user);
+        ticketDao.add(ticket);
+
+        Order order = new Order();
+        final OrderDao orderDao = (OrderDao)
+                injector.getInstance(OrderDao.class);
```

Comment:
Directly using `OrderDao` instead of an `OrderService` might not align with the intended service-oriented architecture. Consider using a service to handle order-related operations.